src/edu/washington/escience/myria/accessmethod/JdbcAccessMethod.java

```diff
@@ -425,6 +425,22 @@ public void dropAndRenameTables(final RelationKey oldRelation, final RelationKey
   }
 
   @Override
+  public void dropTableIfExistsCascade(final RelationKey relationKey) throws DbException {
+    switch (jdbcInfo.getDbms()) {
+      case MyriaConstants.STORAGE_SYSTEM_MYSQL:
+        LOGGER.warn("MySQL does not implement CASCADE");
```

Comment:
I'm not sure that just logging a warning is the right approach since it might never be seen. I think if we're violating the user's expectations then the correct thing to do is throw an exception. This may be all moot for now given that we don't use MySQL, but who knows what external users might want to do...